### PR TITLE
Add `pcntl_unshare` description

### PR DESCRIPTION
**Pull request by saundefined**
Relates to #663

### CODE REVIEW

File: reference/pcntl/functions/pcntl-unshare.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <para>
+   <variablelist>
+    <varlistentry>
+     <term><parameter>flags</parameter></term>
+     <listitem>
+      <para>
+       The <parameter>flags</parameter> parameter is a bitmask that specifies which parts of the execution context should be unshared.
+       This parameter is specified by ORing together zero or more of the <literal>CLONE_*</literal> constants:
```
Comment (cmb69) [inline reference/pcntl/functions/pcntl-unshare.xml:31]: The last character on that line should either be a period, or you may want to list the individual constants.

File: reference/pcntl/constants.xml [hunk 2]
```diff
@@ -965,6 +965,83 @@
     </simpara>
    </listitem>
   </varlistentry>
+  <varlistentry xml:id="constant.clone-newns">
```
Comment (cmb69) [inline reference/pcntl/constants.xml:968]: Maybe add some grouping here (similar to https://www.php.net/manual/en/imagick.constants.php); then later on we could link to that xml:id.

File: reference/pcntl/functions/pcntl-unshare.xml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    </varlistentry>
+   </variablelist>
+  </para>
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns <literal>0</literal> on success, <literal>-1</literal> otherwise.
+   On failure it sets an error code, that can be receive with <function>pcntl_get_last_error</function>.
```
Comment (cmb69) [inline reference/pcntl/functions/pcntl-unshare.xml:43]: Maybe
```suggestion
   On failure it sets an error code, that can be retrieved with <function>pcntl_get_last_error</function>.
```

**Review by cmb69 — COMMENTED** [review]
Thank you! A few nits, but otherwise looks fine (although I don't know about that function).

File: reference/pcntl/functions/pcntl-unshare.xml (hunk 1 above)
Comment (saundefined) [reply reference/pcntl/functions/pcntl-unshare.xml:31]: Oops.. fixed it. I listed the constants below. Or is it better to provide a link to the constants group?

File: reference/pcntl/constants.xml (hunk 2 above)
Comment (saundefined) [reply reference/pcntl/constants.xml:1010]: Thanks! Grouped it.

Constants has xml id `constant.clone-newns`, not `pcntl.constant.clone-newns` — is it ok?
Or is it better to add a prefix?

File: reference/pcntl/functions/pcntl-unshare.xml (hunk 1 above)
Comment (cmb69) [reply reference/pcntl/functions/pcntl-unshare.xml:31]: It's fine this way. :)

File: reference/pcntl/constants.xml (hunk 2 above)
Comment (cmb69) [reply reference/pcntl/constants.xml:1010]: Unless there would be a conflict, `constant.clone-newns` etc. is fine.

**Comment by Girgias** [comment]
I've merged this as one can always improve the documentation afterwards.

Thanks for the contribution!

**Pull request merged**